bin/rad-repl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env radicle
+(load! (find-module-file! "prelude.rad"))
+(import prelude/machine '[eval] :unqualified)
+
+(def help
+  "Default help text."
+  (fn []
+    "radicle is a LISP-like language intended for programming chains, and interacting with those chains.
```

Comment:
Wouldn't it be nicer if help would directly print the help text ? easier to read then.